datafusion/expr/src/expr.rs

```diff
@@ -1663,6 +1667,13 @@ impl Expr {
             | Expr::Placeholder(..) => false,
         }
     }
+
+    pub fn spans(&self) -> Option<&Spans> {
```

Comment:
Can you please add comments about what `Spans` means in this context?

Reply:
Added